lms/static/sass/_header.scss

```diff
@@ -0,0 +1,365 @@
+/*
+  This file contains all the necessary styling for a uniform
+  navigation bar that can exist on any of the v1, v2 or bootstrap
+  pages.
+*/
+.global-header {
+  box-sizing: border-box;
+  width: 100%;
+  overflow: hidden;
+  padding-bottom: $baseline/2;
+  border-bottom: 1px solid theme-color("primary");
+  box-shadow: 0 1px 5px 0 rgba(0, 0, 0, 0.1);
+  background: $white;
+
+  /*
+    Logo and course identification block
+  */
+  .header-logo {
+    display: inline;
+
+    a {
+      @include float(left);
```

Comment:
I'd prefer to use flexbox instead of floats, unless you need text to wrap around the floated element.

Reply:
I'd still like to see this change if it is straightforward.